M365/Search-MailboxExtendedProperty.ps1

```diff
@@ -0,0 +1,145 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+#Requires -Modules @{ ModuleName="ExchangeOnlineManagement"; RequiredVersion="3.4.0" }
+#Requires -Modules @{ ModuleName="Microsoft.Graph.Users"; RequiredVersion="2.24.0" }
+#Requires -Modules @{ ModuleName="Microsoft.Graph.Mail"; RequiredVersion="2.24.0" }
```

Comment:
@cboonham do we require exactly this version or is this the minimum version that is required? If it's the minimum version, it would be better to use `ModuleVersion` instead as this specifies a minimum acceptable version of the module.

Reply:
OK, done. I was undecided whether to do this originally, as it means testing with later versions, but it should be fine.